_episodes/arrays.md

```diff
@@ -421,3 +421,16 @@ which returns a random sample of the values in the list.
 > >
 > {: .solution }
 {: .challenge }
+
+> ## Comments on a static blog
+>
+> Comments section can be included in a static blog using one of the below approaches.
+> 1. Disqus is the first approach. Disqus website offers [instructions](https://disqus.com/admin/install/platforms/jekyll/) on how to enable comments for Jekyll platform (used for GitHub pages). [Demo](https://erresen.github.io/jekyll/disqus/2016/08/26/getting-comments-working-jekyll-disqus.html). Although if you are privacy inclined, the likes of Privacy Badger blocks Disqus widget and might look for alternatives.
+> 2. [Staticman](https://staticman.net/) is the second approach. This setup is tedious and it is easy to get the configuration wrong. [Demo](https://travisdowns.github.io/blog/2020/02/05/now-with-comments.html).
```

Comment:
I would prefer to use more passive language than "tedious" here. Perhaps we could go with something like below?

> The setup is more complex than some of the other options and, in our experience, it is easy to get the configuration wrong.

Reply:
agreed, fixed with 7c58ae